includes/class-wc-referralcandy-integration.php

```diff
@@ -75,26 +95,38 @@ public function woocommerce_order_referralcandy($order_id) {
             $billing_first_name = $wc_pre_30? $order->billing_first_name : $order->get_billing_first_name();
             $billing_last_name  = $wc_pre_30? $order->billing_last_name : $order->get_billing_last_name();
             $billing_email      = $wc_pre_30? $order->billing_email : $order->get_billing_email();
+            $encoded_email      = urlencode($billing_email);
             $order_total        = $order->get_total();
             $order_currency     = $wc_pre_30? $order->get_order_currency() : $order->get_currency();
             $order_number       = $order->get_order_number();
 
             $divData = [
-                'id'                => 'refcandy-mint',
+                'id'                => $this->is_option_enabled('popup')? 'refcandy-popsicle' : 'refcandy-mint',
                 'data-app-id'       => $this->get_option('app_id'),
                 'data-fname'        => urlencode($billing_first_name),
                 'data-lname'        => urlencode($billing_last_name),
-                'data-email'        => urlencode($billing_email),
+                'data-email'        => $this->is_option_enabled('popup')? $billing_email : $encoded_email,
```

Comment:
🤔why don't encoded it all the time?

Reply:
@llun encoded email doesn't work with popsicle / pppw :(